File: TaliExpress.Server/Classes/API/Responses/LoginResponse.cs
using TaliExpress.Server.Classes.AngularObjects;

namespace TaliExpress.Server.Classes.API.Responses
{
    public sealed class LoginResponse : BaseApiResponse
    {
        public UserAng User { get; set; } = new UserAng();
        public StoreDataAng StoreData { get; set; } = new StoreDataAng();
        public CartDataAng CartData { get; set; } = new CartDataAng();
    }
}

File: Taliexpress.client/Classes/ApiResponse/LoginResponse.ts
import { User } from "../Common/User";
import { CartData } from "../Common/CartData";
import { StoreData } from "../Common/StoreData";
import { BaseApiResponse } from "./BaseApiResponse";

export class LoginResponse extends BaseApiResponse {
  user: User = new User();
  cartData: CartData = new CartData();
  storeData: StoreData = new StoreData();
}
